### ISSUE TITLE
Todor performance on 0.80.0-dev

### ISSUE THREAD

**Issue by andraaspar** [nesbox/TIC-80#966]
Hi everyone,

Masternama just made me aware that my game has performance issues on the 0.80.0-dev version of TIC-80. See https://tic.computer/play?cart=972#comment-2115 How would I go about fixing this? Where to get a fresh build of TIC-80? (Ideally on Android?) Is it mature enough to worry about pderformance? Is it an issue with my code or something in the TIC? I know, lots of questions. Can you give me any pointers please?

Thanks!

**Comment by StinkerB06** [comment]
https://builds.tic.computer/master/2019.11.30

**Comment by nesbox** [comment]
@andraaspar Hi, first of all, please, try to compare performance in .70 and .80 by yourself.
You can download the latest build here https://builds.tic.computer/master
Thanks.

**Comment by andraaspar** [comment]
Thank you. I will start looking into this.

**Comment by andraaspar** [comment]
@StinkerB06 @nesbox Thank you. I checked out the latest build, and did not find any issues in the TIC. I looked into my own code instead, and made lots of internal changes and managed to speed it up by roughly 40-60%, making it a lot less likely to drop frames. The performance improvements were: using a lot less Moonscript classes for short lived data objects, reducing the amount of poke calls, and tweaking physics calculations to make them less wasteful. It now runs smoothly even on my mobile! Very happy!
